tensorflow/examples/speech_commands/recognize_commands.py

```diff
@@ -28,16 +28,25 @@ class RecognizeResult(object):
   Attributes:
     founded_command: A string indicating the word just founded. Default value
       is '_silence_'
-    score: An float representing the confidence of founded word. Default
+    score: A float representing the confidence of founded word. Default
       value is zero.
     is_new_command: A boolean indicating if the founded command is a new one
       against the last one. Default value is False.
   """
 
-  def __init__(self):
-    self._founded_command = "_silence_"
-    self._score = 0
-    self._is_new_command = False
+  def __init__(self, founded_command="_silence_",
+               score=0.0, is_new_command=False):
+    """
+    Construct a recognition result
```

Comment:
Style nit: please make sure the docstring comments are whole sentences, with punctuation.
We also prefer imperative style for comments, that is, "Constructs a ..." as opposed to "Construct a ...".

Reply:
Out of curiosity: is this text generated based on the class name?

Reply:
Not sure this isn't my code. On GitHub it only goes back to 4e90ebf

Reply:
Aren't you adding the docstring and default arguments to the constructor in this PR? Or is this a merge artifact?